Extensions/ArtifactEngine/Engine/artifactEngine.ts

```diff
@@ -33,11 +33,11 @@ export class ArtifactEngine {
                     this.logger.logSummary();
                     resolve(this.artifactItemStore.getTickets());
                 }, (err) => {
```

Comment:
is this the only place to catch & log telemetry within artifact-engine?

can you also check in individual tasks to see if they need to log any telemetry?

Reply:
yes all the errors should be propagated back here. for task we will need to add telemetry separately.